canvas_modules/common-canvas/src/common-properties/components/flexible-table/flexible-table.jsx

```diff
@@ -419,10 +420,13 @@ export default class FlexibleTable extends React.Component {
 		let searchBar = null;
 
 		if (typeof this.props.filterable !== "undefined" && this.props.filterable.length !== 0) {
-			const placeHolder = formatMessage(this.props.controller.getReactIntl(),
-				MESSAGE_KEYS.TABLE_SEARCH_PLACEHOLDER) + " " + searchLabel;
-			const searchBarLabel = formatMessage(this.props.controller.getReactIntl(),
-				MESSAGE_KEYS.TABLE_SEARCH_LABEL, { table_name: this.props.tableLabel });
+			const placeHolder = this.props.intl.formatMessage(
```

Comment:
Are we concatenating labels here?

Reply:
Yes, `"table.search.placeholder": "Find in column",`
We show -` Find in column {Column_name}`

Reply:
ok, this should be fix.